packages/synapse-react-client/src/components/CitationPopover/CitationPopover.tsx

```diff
@@ -0,0 +1,325 @@
+            gap: '20px',
+            padding: '20px',
+            width: { xs: '100%', md: '500px' },
+          }}
+        >
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography
+              sx={{ fontSize: '20px', fontWeight: 700, lineHeight: 'normal' }}
+            >
+              Cite As:
+            </Typography>
+            <CloseIcon
+              onClick={handleClose}
+              sx={{
+                display: { xs: 'block', sm: 'none' },
+                marginLeft: 'auto',
+                color: '#878E95',
+              }}
+            />
+          </Box>
+          <Typography
+            variant="smallText1"
+            sx={{
+              lineHeight: 'normal',
+              maxHeight: { xs: '100%', sm: '300px' },
+              overflowY: 'scroll',
+              wordWrap: 'break-word',
+            }}
+          >
+            {boilerplateText}
+            <br />
+            {isLoading ? (
+              <Box sx={{ fontWeight: 700 }}>
+                <br />
+                Loading citation...
+              </Box>
+            ) : (
+              citation
+            )}
+          </Typography>
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: { xs: 'column', sm: 'row' },
+              gap: { xs: '10px', sm: '8px' },
+              width: '100%',
+              maxHeight: '38px',
+            }}
+          >
+            <Box sx={{ flex: 1 }}>
+              <FormControl sx={selectSx}>
+                <Select
+                  variant="standard"
+                  sx={{
+                    svg: {
+                      color: '#878E95',
+                      width: '24px',
+                      height: '24px',
+                      right: '10px',
+                    },
+                  }}
+                  value={citationFormat}
+                  onChange={handleChange}
+                  IconComponent={KeyboardArrowDown}
+                >
+                  <MenuItem value={'bibtex'}>bibtex</MenuItem>
+                  <MenuItem value={'apa'}>apa</MenuItem>
+                  <MenuItem value={'ieee'}>ieee</MenuItem>
+                  <MenuItem value={'nature'}>nature</MenuItem>
+                  <MenuItem value={'science'}>science</MenuItem>
+                </Select>
+              </FormControl>
+            </Box>
+            <Button
+              onClick={() => handleDownload(citation || '')}
+              sx={{
+                flex: 1,
+                padding: '6px 16px',
+                borderRadius: 0,
+                backgroundColor: 'primary.main',
+                color: '#FFF !important',
+                whiteSpace: 'nowrap',
+                '&:hover': {
+                  backgroundColor: darken(theme.palette.primary.main, 0.2),
+                  textDecoration: 'none',
+                },
+              }}
+            >
+              Download Citation
+            </Button>
```

Comment:
Why can't we just use the variant='contained' prop instead of adding all of these custom colors and styles? This feels like it should just be

```suggestion
            <Button
              onClick={() => handleDownload(citation || '')}
              sx={{
                flex: 1,
                whiteSpace: 'nowrap',
              }}
            >
              Download Citation
            </Button>
```